crates/bevy_ecs/src/system/commands/mod.rs

```diff
@@ -512,6 +512,95 @@ impl<'w, 's, 'a> EntityCommands<'w, 's, 'a> {
         self
     }
 
+    /// Moves a [`Bundle`] to a target entity.
```

Comment:
```rust
/// {Moves/Clones} a [`{Trait}`] {to/from} a {target/source} entity.
```

I've picked this for all the documentation across the `MoveBundle` and `CloneBundle` PR. If anyone has some better suggestions I can change it because as is, this is pretty aburpt.